index.html

```diff
@@ -0,0 +1,172 @@
+                          <div class="d-flex flex-column">
+                            <h3>JavaScript</h3>
+                          <p>JavaScript is an object-oriented programming language employed by most websites along with HTML and CSS to create robust, dynamic and interactive user experiences.</p>
+                          </div>
+                        </div>
+                        <div class="about-item">
+                          <a href="#"><img src="imgs/algorithm.jpg" alt="about-image1" height="84"></a>
+                          <div class="d-flex flex-column">
+                            <h3>Algorithm</h3>
+                          <p>Learn how to structure and use algorithms to solve real life problems.</p>
+                          </div>
+                        </div>
+                      </div>
+                      <a href="#" class="d-block text-white text-center my-5 fs-5">SEE THE COMPLETE COURSES</a>
+                    </div>
+                  </div>
+                </div>
+            </div>
+        </section>
+        <section id="about-section1">
+            <div class="container-fluid about-page1 p-0">
+                <div class="container-md">
+                  <div class="row">
+                    <div class="col-12 p-0 m-0">
+                      <article class="mt-3 pt-5 about-section">
+                        <h2 class="mt-4 py-2">Partner</h2>
+                        <div class="my-4 about-line"></div>
+                      </article>
+                      <div class="about-item1">
+                        <a href="#"><img src="imgs/naver.png" alt="about-image1" height="84"></a>
+                        <div class="d-flex flex-column">
+                        </div>
+                      </div>
+                      <div class="about-item1">
+                        <a href="#"><img src="imgs/duamkako.png" alt="about-image1" height="84"></a>
+                        <div class="d-flex flex-column">
+                        </div>
+                      </div>
+                      <div class="about-item1">
+                        <a href="#"><img src="imgs/Google.png" alt="about-image1" height="84"></a>
+                        <div class="d-flex flex-column">
+                        </div>
+                      </div>
+                      <div class="about-item1">
+                        <a href="#"><img src="imgs/airbnb.png" alt="about-image1" height="84"></a>
+                        <div class="d-flex flex-column">
+                        </div>
+                      </div>
+                      <div class="about-item1">
+                        <a href="#"><img src="imgs/mozilla.png" alt="about-image1" height="84"></a>
+                        <div class="d-flex flex-column">
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Please make your partner section look like the one provided:

Your current design:

![image](https://user-images.githubusercontent.com/38522578/137812712-e4e3a596-2bf5-4bea-9c5d-c7e3339841f3.png)

Provided design (don't forget to adapt it to the mobile version too):

![image](https://user-images.githubusercontent.com/38522578/137812803-635874bd-7bf6-485b-99f2-0ac6cf64928c.png)